Client/Frontend/BraveVPN/BraveVPNSettingsViewController.swift

```diff
@@ -141,6 +148,21 @@ class BraveVPNSettingsViewController: TableViewController {
     
     deinit {
         NotificationCenter.default.removeObserver(self)
+        serverListRequest?.cancel()
+        serverListRequest = nil
```

Comment:
Redundant setting nil on deinit

Reply:
Interesting, i use this pattern in most async things I do, timers, dispatchWorkItems, dataTasks, as extra safety step,
will fix it